src/server/HtmlDocument.js

```diff
@@ -227,6 +230,19 @@ class HtmlDocument extends React.Component {
             name="msapplication-TileImage"
             content="/static/favicons/mstile-144x144.png"
         />
+
+        <script dangerouslySetInnerHTML={{__html: envConfig}} />
+        { GA_TRACKING_CODE &&
+            <script dangerouslySetInnerHTML={{__html: `
+                window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+                    ga('create', 'UA-XXXXX-Y', 'auto');
+                    ga('send', 'pageview');
+            `}} />
+        }
+        { GA_TRACKING_CODE &&
+            <script async src='//www.google-analytics.com/analytics.js'></script>
```

Comment:
This is confusing. Break it all out into a function called `renderAnalyticsBlock()` or something.